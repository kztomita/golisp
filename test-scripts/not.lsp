(print (not nil))
(print (not ()))
(print (not '()))
(print (not 1))
(print (not 0))
(setq a nil)
(print (not a))
