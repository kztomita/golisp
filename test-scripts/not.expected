t
t
t
nil
nil
t
